AD-reviews/review-ietf-dnssd-prireq-04-intdir-lc-halley-2020-01-30.txt

```diff
@@ -0,0 +1,16 @@
+* review-ietf-dnssd-prireq-04-intdir-lc-halley-2020-01-30
+
+I am an assigned INT directorate reviewer for draft-ietf-dnssd-prireq.  These
+comments were written primarily for the benefit of the Internet Area Directors.
+Document editors and shepherd(s) should treat these comments just like they
+would treat comments from any other IETF contributors and resolve them along
+with any other Last Call comments that have been received. For more details on
+the INT Directorate, see http://www.ietf.org/iesg/directorate.html.
+
+The draft does a very good job explaining the problem space and its
+complexities.  The guidance it offers for future extensions is sound.
+
+/Bob
+
+P.S. I liked the ASCII art.
+
```

Comment:
Same as above. We should not add a file per review in the depot. Open issues instead.